xwiki-platform-core/xwiki-platform-livetable/xwiki-platform-livetable-ui/src/main/resources/XWiki/LiveTableResultsMacros.xml

```diff
@@ -30,8 +30,8 @@
   <parent>XWiki.WebHome</parent>
   <author>xwiki:XWiki.Admin</author>
   <contentAuthor>xwiki:XWiki.Admin</contentAuthor>
-  <date>1465822369000</date>
-  <contentUpdateDate>1465822369000</contentUpdateDate>
+  <date>1465895028000</date>
```

Comment:
FTR, I agree with Sergiu. I'm a bit surprised since I was pretty sure this was the best practice. At least that's what I always do (and what I've seen the others do).